core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -814,85 +810,86 @@
    * Rocord size in case of compaction.
    */
   public static final int COMPACTION_INMEMORY_RECORD_SIZE = 120000;
-
-  /**
-   * If the level 2 compaction is done in minor then new compacted segment will end with .2
-   */
-  public static String LEVEL2_COMPACTION_INDEX = ".2";
-
-  /**
-   * Indicates compaction
-   */
-  public static String COMPACTION_KEY_WORD = "COMPACTION";
-
   /**
    * hdfs temporary directory key
    */
   public static final String HDFS_TEMP_LOCATION = "hadoop.tmp.dir";
-
   /**
    * zookeeper url key
    */
   public static final String ZOOKEEPER_URL = "spark.deploy.zookeeper.url";
-
   /**
    * configure the minimum blocklet size eligible for blocklet distribution
    */
   public static final String CARBON_BLOCKLETDISTRIBUTION_MIN_REQUIRED_SIZE =
       "carbon.blockletdistribution.min.blocklet.size";
-
   /**
    * default blocklet size eligible for blocklet distribution
    */
   public static final int DEFAULT_CARBON_BLOCKLETDISTRIBUTION_MIN_REQUIRED_SIZE = 2;
-
+  /**
+   * This batch size is used to send rows from load step to another step in batches.
+   */
+  public static final String DATA_LOAD_BATCH_SIZE = "DATA_LOAD_BATCH_SIZE";
+  /**
+   * Default size of data load batch size.
+   */
+  public static final String DATA_LOAD_BATCH_SIZE_DEFAULT = "1000";
+  /**
+   * carbon data file version property
+   */
+  public static final String CARBON_DATA_FILE_VERSION = "carbon.data.file.version";
+  /**
+   * current data file version
+   */
+  public static final short CARBON_DATA_FILE_CURRENT_VERSION = 2;
```

Comment:
it is not easy to understand what current means, can you change to a more meaning name, and use enum for version number.
The default version is 1 or 2?